### DISCUSSION TITLE
How to move outputs up one directory?

### DISCUSSION THREAD

**Discussion by StevenTCramer** [statiqdev/Discussions#172] [Q&A] [answered]
I want to group some input into a folder but don't want that folder in the output.  What is the easiest way to do that?

I thought `_directory.yml` would be how to do it but I can't figure out the right incantation.

**Comment by StevenTCramer** [comment, marked as answer]
I got the following to do what I wanted.

changing 
`pages/components/capturing-unexpected-parameters/`
to
`components/capturing-unexpected-parameters/`

```yaml
DestinationPath: 
  =>
  { 
    List<string> extensions = new List<string> {"html","cshtml","md"};
    var pagesPath = new NormalizedPath("pages");
    var relativePath = pagesPath.GetRelativePath(Document.Destination);
    Context.LogDebug($"relativePath:{relativePath}");
    if(extensions.Any(str => str == Document.Destination.Extension))
    {
      var newDestination = relativePath.ChangeExtension("html");
      return newDestination;
    } else
    {
      return relativePath;
    }
  }
```

Maybe there is a cleaner way?

**Reply by daveaglick** [reply]
Beat me to it :) (I'm obviously running a little slow with discussion responses lately 😬)

While it's pretty easy to exclude an entire folder using the directory metadata and changing content/media types, that's not quite what you're after. What you have here, or something quite similar, is probably the best way. Putting code like this in the `_directory.yml` file is probably _possible_ using [computed metadata values](https://www.statiq.dev/guide/documents-and-metadata/metadata-values#computed-values) for the destination value but it would get pretty messy and hard to debug.
